core/dbt/events/functions.py

```diff
@@ -171,6 +172,22 @@ def create_text_log_line(e: T_Event, msg_fn: Callable[[T_Event], str]) -> str:
     return log_line
 
 
+def create_file_text_log_line(e: T_Event, msg_fn: Callable[[T_Event], str]) -> str:
```

Comment:
What do you think about:
- padding all thread names to be equal width (up to 10 characters, the length of `MainThread`)
- including `level` in the file logger — I imagine it might be helpful to scan/search for `error` + `warn` messages

Reply:
I empathize with the desire to make them even. It would be slightly better to read. But there is also a thread name ThreadPoolExecutor-0_0.  Should I just truncate that one?

Reply:
I truncated ThreadPoolExecutor-0_0 to ThreadPool, and changed to the square brackets. Making the prefix consistent does help with the reading. Still not entirely happy with the padded spaces in the thread name. We could translate MainThread to 'Thread-0' and ThreadPool to 'Thread-P'. Is it possible for the thread number to go higher than 9?

Anyway, take a look and see what you think

Reply:
I definitely like this more than the current version!

Switch to square brackets is good. Spacing is a teensy bit awkward, but I do think it makes it easier to read.

> Is it possible for the thread number to go higher than 9?

In theory, users can set it as high as they want, but I don't think we should worry about it being 3+ digits, let alone >4